controllers/useraccount/useraccount_controller_test.go

```diff
@@ -416,16 +414,15 @@ func TestReconcile(t *testing.T) {
 		cfg := commonconfig.NewMemberOperatorConfigWithReset(t)
 
 		mockCallsCounter := new(int)
-		userAcc := newUserAccount(username, userID)
-		util.AddFinalizer(userAcc, toolchainv1alpha1.FinalizerName)
+		userAcc := newUserAccount(username, userID, withFinalizer())
 		r, req, cl, _ := prepareReconcile(t, username, cfg, userAcc, preexistingUser, preexistingIdentity)
 
 		t.Run("first reconcile deletes identity", func(t *testing.T) {
 			// given
 			// Set the deletionTimestamp
-			now := metav1.NewTime(time.Now())
-			userAcc.DeletionTimestamp = &now
-			err = r.Client.Update(context.TODO(), userAcc)
+			//now := metav1.NewTime(time.Now())
+			//userAcc.DeletionTimestamp = &now
```

Comment:
```suggestion
```

Reply:
done in [0b5289e](https://github.com/codeready-toolchain/member-operator/pull/551/commits/0b5289e17f8337c2eeba756dd94a524b87163fd0), thanks